core/include/core/G3NetworkSender.h

```diff
@@ -45,7 +45,7 @@ class G3NetworkSender : public G3Module {
 			std::lock_guard<std::mutex> lg(lock);
 			// Refuse to put anything into stopped queues, and if a maximum
 			// queue size is set and would be exceeded, drop low priority data
-			if (die || (max_size && low_priority && queue.size()>=max_size))
+			if (die || (max_size && low_priority && queue.size()>=(size_t)max_size))
```

Comment:
Myb we should just change `max_size` to be `size_t` in the first place? E.g. there's no reason to allow it to be negative.

Reply:
It's a user argument, so I figured that a negative value might mean something specific, but maybe that's not the case.

Reply:
It would technically do something (never allow anything 'low priority' to be put in the queue), but it seems doubtful that anyone wanted to do that.